hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1718,6 +1718,20 @@
         "chip_sw_otp_ctrl_lc_signals_rma",            // lc_seed_hw_rd_en_i, lc_creator_seed_sw_rw_en_i, lc_keymgr_en_i: otp_ctrl
       ]
     }
+    {
```

Comment:
thanks @sriyerg 
if you havent done so already, would you mind filing the corresponding github issues? 
it would be good to do it now so we don't forget about it later.

Reply:
Yes - I was going to do that after getting this PR reviewed.

Reply:
Done - #16071, #16069, #16070.